diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3cf11fc..c9e9e18 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -177,6 +177,7 @@ set (HEADERS
    ${SOURCE_DIR}/src/mongoc/mongoc-opcode.h
    ${SOURCE_DIR}/src/mongoc/mongoc-opcode-private.h
    ${SOURCE_DIR}/src/mongoc/mongoc-read-concern.h
+   ${SOURCE_DIR}/src/mongoc/mongoc-read-concern-private.h
    ${SOURCE_DIR}/src/mongoc/mongoc-read-prefs.h
    ${SOURCE_DIR}/src/mongoc/mongoc-server-description.h
    ${SOURCE_DIR}/src/mongoc/mongoc-socket.h
@@ -379,3 +380,26 @@ install(
   FILES ${HEADERS}
   DESTINATION "include/libmongoc-${MONGOC_API_VERSION}"
 )
+
+# Define pkg-config files
+set(VERSION "${MONGOC_VERSION}")
+set(prefix "${CMAKE_INSTALL_PREFIX}")
+set(libdir "\${prefix}/lib")
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/libmongoc-1.0.pc.in
+  ${CMAKE_CURRENT_BINARY_DIR}/libmongoc-1.0.pc
+@ONLY)
+install(
+  FILES ${CMAKE_CURRENT_BINARY_DIR}/libmongoc-1.0.pc
+  DESTINATION lib/pkgconfig
+)
+if (MONGOC_ENABLE_SSL)
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/libmongoc-ssl-1.0.pc.in
+    ${CMAKE_CURRENT_BINARY_DIR}/libmongoc-ssl-1.0.pc
+  @ONLY)
+  install(
+    FILES ${CMAKE_CURRENT_BINARY_DIR}/libmongoc-ssl-1.0.pc
+    DESTINATION lib/pkgconfig
+  )
+endif()
