diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4202c00..cb8af46 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,9 +1,11 @@
 add_custom_target(install_headers
     COMMAND "${CMAKE_COMMAND}" -DCMAKE_INSTALL_COMPONENT=dev -P "${CMAKE_BINARY_DIR}/cmake_install.cmake"
+    DEPENDS bsoncxx mongocxx
 )
 
 add_custom_target(install_libs
     COMMAND "${CMAKE_COMMAND}" -DCMAKE_INSTALL_COMPONENT=runtime -P "${CMAKE_BINARY_DIR}/cmake_install.cmake"
+    DEPENDS bsoncxx mongocxx
 )
 
 add_subdirectory(bsoncxx)
